docs/develop/09_polarpath/01_overview.md

```diff
@@ -0,0 +1,91 @@
+---
+id: what-is-polarpath
+title: Overview
+---
+
+Polar path consists of an `asset-swap` pallet for KILT chain runtimes and extrinisics that let parachain-based chain developers to make their native token accessible on the Ethereum network using [the Snowbridge bridge](https://docs.snowbridge.network) between Polkadot and Ethereum.
+
+Polar path provides parachains with a solution for creating an ERC-20 wrapper around their native tokens, enabling a trustless conversion of any existing native parachain token (PARA) into a wrapped parachain token (wPARA) on Ethereum while maintaining the trustlessness guarantees Snowbridge provides.
+
+The pallet enables a configured conversion rate of parachain tokens on the origin chain to an ERC-20 wrapper token on Polkadot's AssetHub.
+
+## Setup
+
+:::tip Asset Hub
+The Polkadot Relay Chain doesn't natively support assets apart from DOT. Instead, parachains support this functionality, and for Polkadot, this parachain is the [Asset Hub](https://wiki.polkadot.network/docs/build-integrate-assets).
+:::
+
+Making a Parachain Token to be usable on Ethereum requires the following steps:
+
+1. Deploy an ERC-20 contract token to Ethereum with a fixed supply based on the parachain needs.
+2. [Disable the token issuer rights](https://ethereum.org/en/guides/how-to-revoke-token-access/) on the ERC-20 contract.
+3. [Register the ERC-20 token on Asset Hub](https://wiki.polkadot.network/docs/learn-assets#creation-and-management).
+4. The entity holding ERC-20 funds performs a one-time transfer of all tokens from their Ethereum account to the parachain's sovereign account on Asset Hub.
```

Comment:
Minus the amount of tokens that he previously locked on KILT. Those will stay under an account of his choice on Etheruem.

Reply:
I assume you're referring to point 4?

Questions…

1. Where in the flow does this locking happen? I don't seem to have seen that in any source material.
2. Are these tokens locked on KILT even when other chains use the pallets?

```suggestion
4. The entity holding ERC-20 funds performs a one-time transfer of all tokens from their Ethereum account minus the amount of tokens previously locked with KILT to the parachain's sovereign account on Asset Hub.
```

Reply:
1. The requirement in the pallet is that before a switch pair can be created, the account for that pair must have sufficient balance. This means that someone has to send enough KILTs to this account. It's not specified anywhere other than the code, which is why I think it's extremely important to describe the process.
2. No, each chain would hold their native token locked in a poo-specific account. KILT will only lock KILTs.